sklearn/ensemble/_forest.py

```diff
@@ -1552,10 +1552,13 @@ class ExtraTreesClassifier(ForestClassifier):
         <n_jobs>` for more details.
 
     random_state : int, RandomState instance or None, optional (default=None)
-        If int, random_state is the seed used by the random number generator;
-        If RandomState instance, random_state is the random number generator;
-        If None, the random number generator is the RandomState instance used
-        by `np.random`.
+        Controls 3 sources of randomness:
```

Comment:
Rendering is a bit weird, I think you need to skip a line after this one


see https://81655-843222-gh.circle-artifacts.com/0/doc/modules/generated/sklearn.ensemble.ExtraTreesClassifier.html

Reply:
Thank you Nicolas. How did you generate the preview?

Reply:
https://scikit-learn.org/dev/developers/contributing.html#generated-documentation-on-circleci

but you can of course also build the docs locally